www/apps/book/app/learn/fundamentals/modules/db-operations/page.mdx

```diff
@@ -683,3 +683,146 @@ class BlogModuleService {
   }
 }
 ```
+
+Here are examples of typical operations using a repository:
+
+```ts
+// Create a new blog post
+const post = await postRepository.create({
+  title: "Introduction to Medusa",
+  content: "Medusa is a composable commerce platform...",
+  author_id: "author_123",
+  status: "draft"
+})
+
+// Find a specific post
+const post = await postRepository.findOne({
+  where: { id: "post_123456" }
+})
+
+// Update a post
+await postRepository.update("post_123456", {
+  status: "published",
+  metadata: {
+    featured: true
+  }
+})
+
+// Delete a post
+await postRepository.delete("post_123456")
```

Comment:
Needs an update to match the delete reference: https://docs.medusajs.com/resources/data-model-repository-reference/methods/delete